karma.conf.js

```diff
@@ -27,23 +27,43 @@ module.exports = function(config) {
 
       'jujugui/static/gui/build/app/assets/javascripts/handlebars.runtime.js',
 
-      'jujugui/static/gui/build/app/components/**/*.js',
-      'jujugui/static/gui/build/**/test-*.js'
+      'jujugui/static/gui/src/app/components/**/*.hbs',
+      'jujugui/static/gui/src/app/components/**/*.js'
     ],
 
 
     // list of files to exclude
     exclude: [
-      'jujugui/static/gui/build/app/components/**/*-min.js',
-      'jujugui/static/gui/build/**/test-*-min.js',
-      'jujugui/static/gui/build/app/assets/javascripts/yui/node_modules/**/*',
-      'jujugui/static/gui/build/app/assets/javascripts/yui/yui/node_modules/**/*'
```

Comment:
I think we'll probably need these if we're doing anything with code outside of the components dir, right?